src/types/index.ts

```diff
@@ -1,9 +1,65 @@
 // @ts-ignore
 interface AutoComplete {}
 
-interface StopGroup {}
+interface StopGroup {
+  country_code: string;
+  name: string;
+  longitude: number;
+  latitude: number;
+  type: string;
+  childs: SegmentProviderStop[];
+  serviced?: string;
+  has_been_modified?: boolean;
+  warning?: boolean;
+  is_latest?: boolean;
+}
+
+interface StopCluster {
+  country_code: string;
+  name: string;
+  longitude: number;
+  latitude: number;
+  type: string;
+  childs: string[];
+  unique_name?: string;
+  serviced?: string;
+  has_been_modified?: boolean;
+  warning?: boolean;
+  is_latest?: boolean;
+}
 
-interface StopCluster {}
+type GroundPlacesList = {
+  [key: string]: StopGroup | StopCluster;
```

Comment:
key:GPUID

Reply:
Can't type key with a custom type here

Reply:
why that ?

Reply:
![Capture d’écran 2020-11-06 à 17 37 07](https://user-images.githubusercontent.com/19270653/98391260-b623ae00-2056-11eb-81bc-a22bf273ca87.png)

Reply:
Update